components/04-views/registration/registration.hbs

```diff
@@ -1,48 +1,58 @@
-
-        <h1 class="registration__title">
-            {{ title }}
-        </h1>
-
-        <div class="registration__content">
-            <fieldset class="fieldset registration__fieldset {{ personalSection.fieldset.class }}"
-                {{{ personalSection.fieldset.attributes }}}
-            >
-                <legend class="fieldset__legend registration__legend {{ personalSection.legend.class }}"
-                    {{{ personalSection.legend.attributes }}}
+    <div class="container">
+        {{ render '@heading' mainHeading }}
+        <div class="registration">
+            <form class="registration__content">
+                <fieldset
+                    class="
+                        fieldset registration__fieldset
+                        {{ personalSection.fieldset.class }}
+                    "
+                    {{ personalSection.fieldset.attributes }}
                 >
-                    {{ personalSection.legend.text }}
-                </legend>
-
-                {{ render '@input' firstName }}
-                {{ render '@input' lastName }}
-                {{ render '@checkbox' newsletterCheckbox }}
-
-            </fieldset>
-
-            <fieldset class="fieldset registration__fieldset {{ signInSection.fieldset.class }}"
-                    {{{ signInSection.fieldset.attributes }}}
-            >
-                <legend class="fieldset__legends registration__legend {{ signInSection.legend.class }}"
-                    {{{ signInSection.legend.attributes }}}
+                    <legend
+                        class="
+                            fieldset__legend registration__legend
+                            {{ personalSection.legend.class }}
+                        "
+                        {{ personalSection.legend.attributes }}
+                    >
+                        {{ personalSection.legend.text }}
+                    </legend>
+                    {{ render '@divider' headingDivider }}
+                    {{ render '@input' firstName }}
+                    {{ render '@input' lastName }}
+                    {{ render '@checkbox' newsletterCheckbox }}
+                </fieldset>
+                <fieldset
+                    class="
+                        fieldset registration__fieldset
+                        {{ signInSection.fieldset.class }}
+                    "
+                    {{ signInSection.fieldset.attributes }}
                 >
-                    {{ signInSection.legend.text }}
-                </legend>
-
-                {{ render '@input' email }}
-                <div class="registration__input">
-                    {{ render '@input' password }}
-                    {{ render '@password-strength' passwordStrength }}
-                </div>
-                {{ render '@input' passwordConfirmation }}
+                    <legend
+                        class="
+                            ieldset__legends registration__legend
```

Comment:
Please fix the class name or remove if it's not used

Reply:
fixed